pkgs/applications/science/misc/sumo/default.nix

```diff
@@ -0,0 +1,56 @@
+{ bzip2, cmake, eigen, fetchFromGitHub, ffmpeg, fox_1_6, gdal, git, gl2ps, gpp
+, gtest, jdk, libGL, libGLU, libX11, libjpeg, libpng, libtiff, openscenegraph
+, proj, python3, python37Packages, stdenv, swig, xercesc, xorg, zlib }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+
+  pname = "sumo";
+  version = "1.8.0";
+
+  src = fetchFromGitHub {
+    owner = "eclipse";
+    repo = "sumo";
+    rev = "4820708f3db155db9001523cac05803ceadceeff";
+    sha256 = "1w9im1zz8xnkdwmv4v11kn1xcqm889268g1fw4y2s9f6shi41mxx";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [
+    bzip2
+    eigen
+    ffmpeg
+    fox_1_6
+    gdal
+    git
+    gl2ps
+    gpp
+    gtest
+    jdk
+    libGL
+    libGLU
+    libX11
+    libjpeg
+    libpng
+    libtiff
+    openscenegraph
+    proj
+    python37Packages.setuptools
+    swig
+    xercesc
+    xorg.libXcursor
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXft
+    xorg.libXrandr
+    xorg.libXrender
+    zlib
+    python3
+  ];
+
+  nativeBuildInputs = [ cmake ];
+
+  enableParallelBuilding = true;
```

Comment:
this should be the default

Reply:
I removed `enableParallelBuilding`, but removing the other option disables building.